pkg/services/libraryelements/models.go

```diff
@@ -170,7 +170,7 @@ type CreateLibraryElementCommand struct {
 type patchLibraryElementCommand struct {
 	FolderID int64           `json:"folderId" binding:"Default(-1)"`
 	Name     string          `json:"name"`
-	Model    json.RawMessage `json:"model"`
+	Model    json.RawMessage `json:"model,omitempty"`
```

Comment:
Without `omitempty` JSON marshaller writes `null` and unmarshaller gets 4 bytes (`[]byte{'n', 'u', 'l', 'l'}`) and most of the tests go bonkers.